src/main/scala/org/pac4j/http4s/Http4sCacheSessionStore.scala

```diff
@@ -0,0 +1,94 @@
+package org.pac4j.http4s
+
+import java.util.UUID
+
+import org.pac4j.core.context.session.SessionStore
+import org.pac4j.core.context.{Cookie, Pac4jConstants}
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
+
+class Http4sCacheSessionStore extends SessionStore[Http4sWebContext] {
```

Comment:
It is just an in-memory map session store, isn't it?

Reply:
Correct. Added more info in doc comments.